crates/cli_bin/Cargo.toml

```diff
@@ -56,5 +57,16 @@ grit_tracing = [
 docgen = ["marzano-cli/docgen"]
 
 [package.metadata.dist]
+# this should be published by cargo-dist (needed because publish = false)
+dist = true
+# build flags for published builds
 default-features = false
 features = ["marzano-cli/grit_beta"]
+# the npm package should have this name
+npm-package = "launcher2"
```

Comment:
Would we want this to be `@getgrit/launcher` to replace our existing npm packge?

Reply:
Yeah, I just wasn't sure if you'd want to do a few test runs on a separate package name first -- but we can just set the final one now.

Reply:
i've updated it to "launcher" now